Update harness-platform-faqs.md (#7271)
harness/developer-hub: kb/platform/harness-platform-faqs.md

This error indicates that you've reached the maximum amount of API keys available for your user. Harness has a limit of 5 API keys per user. To resolve this error, delete one of your existing API keys or use a service account instead.

> ### Is there an api call to fetch api-key?

Reviewer: ```suggestion
### Is there an API call to fetch an API key?
```

> No there is no api to get api-key value as Harness does not store the API token for future reference.

Reviewer: ```suggestion
There's no API to get an api-key value as Harness does not store the API token for future reference.
```

You can find this value on the Authentication page (right below the Overview menu on the left). The field will be at the bottom of the page: ```Session Inactivity Timeout (in minutes)```.

> ### Ldap sync is not working for some groups, while for other groups it's working fine, and even manual sync is not helping.

Reviewer: ```suggestion
### Why is LDAP sync not working for some groups despite working fine for others, and even manual sync isn't helping?
```
